### PR TITLE
Featurling for store

### PR DESCRIPTION
**Pull request by colindamelio**
Please review my code :)
Be thoughtful, constructive, and positive.

### CODE REVIEW

File: scripts.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
+const storeApp = {};
```
Comment (genc1499) [inline scripts.js:1]: I like how the app is namespaced

File: scripts.js [hunk 2]
```diff
@@ -1,84 +1,110 @@
...
+    const listItem = document.createElement('li');
+
+    const title = document.createElement('h2');
+    title.textContent = item.title;
+
+    const image = document.createElement('img');
+    image.src = item.url;
 
-  // STEP ONE: filter the inventory so that only items with images that are also in stock are displayed
+    const price = document.createElement('p')
```
Comment (rslmason) [inline scripts.js:82]:
```suggestion
    const price = document.createElement('p');
```

File: scripts.js [hunk 3]
```diff
@@ -44,86 +44,67 @@ const totalInventory = [
   },
```
Comment (genc1499) [inline scripts.js:44]: remove comma

Comment (genc1499) [inline scripts.js:44]: remove comma

File: scripts.js [hunk 4]
```diff
@@ -1,84 +1,110 @@
+const storeApp = {};
```
Comment (genc1499) [inline scripts.js:1]: good use of namespacing

**Pull request closed without merging**